handbook/repository/generate.md

```diff
@@ -2,7 +2,7 @@
 
 ## Generate a component
 
-To generate a new component in island-ui-core.
+To generate a new React component in island-ui-core.
```

Comment:
Could you at the same time fix the following line (L50; just under)

from:
```
yarn generate @nrwl/node:lib my-lib
```

to:
```
yarn generate @nrwl/nest:lib my-lib
```

Reply:
Changed+1